Src/ST-LIB.cpp

```diff
@@ -21,6 +21,7 @@ void STLIB::start(string ip, string subnet_mask, string gateaway,  UART::Periphe
 }
 
 void STLIB::update() {
-    Ethernet::update();
+    //Ethernet::update();
 	ErrorHandlerModel::ErrorHandlerUpdate();
+	SPI::packet_update();
```

Comment:
I believe this should be left up to the user to place it there, I understand that by default the map is empty, yet it bloats the function when in reality it is doing nothing